src/java/org/apache/cassandra/service/reads/repair/BlockingReadRepairs.java

```diff
@@ -33,14 +33,14 @@
 import org.apache.cassandra.schema.TableMetadata;
 import org.apache.cassandra.tracing.Tracing;
 
+import static org.apache.cassandra.config.CassandraRelevantProperties.DROP_OVERSIZED_READ_REPAIR_MUTATIONS;
 import static org.apache.cassandra.db.IMutation.MAX_MUTATION_SIZE;
 
 public class BlockingReadRepairs
 {
     private static final Logger logger = LoggerFactory.getLogger(BlockingReadRepairs.class);
 
-    private static final boolean DROP_OVERSIZED_READ_REPAIR_MUTATIONS =
-        Boolean.getBoolean("cassandra.drop_oversized_readrepair_mutations");
+    private static final boolean DROP_OVERSIZED_READ_REPAIR_MUTATIONS_PROP = DROP_OVERSIZED_READ_REPAIR_MUTATIONS.getBoolean();
```

Comment:
should not this be named `DROP_OVERSIZED_READ_REPAIR_MUTATIONS_VALUE` ?

PROP -> VALUE. That thing is not "property". It is its value.

Reply:
Renamed as suggested.